lms/djangoapps/learner_dashboard/urls.py

```diff
@@ -0,0 +1,11 @@
+"""
+Learner's Dashboard urls
+"""
+
+from django.conf.urls import patterns, url
+from django.conf import settings
+
+urlpatterns = patterns(
+    '',
+    url(r'^programs', 'learner_dashboard.views.view_programs', name="get_program_listing_view")
```

Comment:
You're also using an old format for your URLconf here. The new syntax looks like [this](https://docs.djangoproject.com/en/1.9/topics/http/urls/#example).